module/modmanager/manager.go

```diff
@@ -770,6 +774,13 @@ func (m *module) checkReady(ctx context.Context, parentAddr string, logger loggi
 			m.handles, err = modlib.NewHandlerMapFromProto(ctx, resp.Handlermap, &m.conn)
 			return err
 		}
+
+		select {
+		case <-slowTicker.C:
+			elapsed := time.Since(startTime).Seconds()
+			logger.Warnf("waiting %q for module client to be ready. Elapsed %.2f seconds", m.cfg.Name, elapsed)
```

Comment:
```suggestion
			logger.Warnf("waiting %q for module to be ready. Elapsed %.2f seconds", m.cfg.Name, elapsed)
```
Module client isn't _really_ a thing; don't want to confuse module authors.